web/packages/teleport/src/Discover/Kubernetes/EnrollEKSCluster/EnrollEksCluster.tsx

```diff
@@ -218,7 +218,12 @@ export function EnrollEksCluster(props: AgentStepProps) {
         emitErrorEvent(
           'unknown error: no results came back from enrolling the EKS cluster.'
         );
-      } else if (result.error) {
+      } else if (
+        result.error &&
+        !result.error.message.includes(
+          'teleport-kube-agent is already installed'
+        )
```

Comment:
does this have any specific error code we could check for instead?

Reply:
Checking for error code as Michael suggested is ideal, but if we're going to check for a substring in an error message, we should make sure to leave a note where the error message in defined in the backend saying that it needs to be kept in sync with this.

Reply:
Not really, it's not an AWS error, just our internal message.

Reply:
I've added a comment 👍